content/docs/operating/configuration.md

```diff
@@ -317,20 +317,29 @@ create multiple target groups:
 
 * One for all API servers, with each API server as a target
 (`__meta_kubernetes_role=apiserver`)
+    * The default port is that used for the API server itself.
 * One for all nodes, with each node as a target (`__meta_kubernetes_role=node`)
+    * The default port is the Kubelet Port from the Kubernetes configuration.
 * One for all services, with each service as a target
 (`__meta_kubernetes_role=service`)
+    * The default port is selected in an undefined manner. If you have multiple
+    ports on your service, you should explicitly configure relabeling to
+    indicate the proper port.
 * One for *each* service, with each service endpoint as a target
 (`__meta_kubernetes_role=endpoint`)
+    * The default port is selected in an undefined manner. If you have multiple
+    ports on your service, you should explicitly configure relabeling to
+    indicate the proper port.
 * One for all pods, with each pod as a target (`__meta_kubernetes_role=pod`)
+    * The default port is the numerically lowest port on the container that is
```

Comment:
Perhaps s/on/exposed by/ ? I seem to not like the "on" in connection with ports. :)